src/dataflow-types/src/client/controller/storage.rs

```diff
@@ -505,6 +533,7 @@ where
                 let changes = collection.read_capabilities.update_iter(update.drain());
                 update.extend(changes);
                 storage_net.push((key, update));
+                if key.is_user() {}
```

Comment:
Is this a left-over?

Reply:
Yes, this has been open for so long that I'm sure there's more. Thanks for catching these! 😃

Reply:
I'm afraid I cannot contribute on a much higher level than this, but I'll continue flagging things that I notice!